lib/sdf-server/src/service/v2/audit_log/list_audit_logs.rs

```diff
@@ -12,16 +12,24 @@ use si_frontend_types as frontend_types;
 use super::AuditLogResult;
 use crate::extract::{AccessBuilder, HandlerContext, PosthogClient, QueryWithVecParams};
 
-#[derive(Deserialize, Serialize, Debug)]
+#[remain::sorted]
+#[derive(Deserialize, Debug)]
+pub enum UserFilter {
+    System,
+    #[serde(untagged)]
+    User(UserPk),
```

Comment:
This is how we handle `string[]` with the "System" string or a `ChangeSetId` conversion to an enum